uk_bin_collection/uk_bin_collection/councils/WestSuffolkCouncil.py

```
@@ -61,7 +61,15 @@ def panel_search(cur_tag: Tag):
 
                     # Parse the date
                     next_collection = date_parse(collection_date)
-                    next_collection = next_collection.replace(year=datetime.now().year)
+
+                    # if current month is December and next_collection month is January, set year to next year
+                    year = datetime.datetime.now().year
```

Ruff (0.14.10)

`datetime` may be undefined, or defined from star imports

(F405)